packages/task/src/browser/task-frontend-contribution.ts

```diff
@@ -211,9 +211,15 @@ export class TaskFrontendContribution implements CommandContribution, MenuContri
             order: '3'
         });
 
-        menus.registerMenuAction(TerminalMenus.TERMINAL_TASKS, {
+        menus.registerMenuAction(TerminalMenus.TERMINAL_TASKS_INFO, {
+            commandId: TaskCommands.TASK_TERMINATE.id,
+            label: 'Terminate Task...',
+            order: '0'
```

Comment:
We should also verify if there is some tasks running and put the sensitivity on the menu item only when there is one or more tasks running

Reply:
We cannot since getting the list of running tasks is an `async` call which the menu `isEnabled` and `isVisible` does not support :(